controllers/api/comment-routes.js

```diff
@@ -0,0 +1,110 @@
+const router = require("express").Router();
+
+const { User, Post, Comment } = require("../../models");
+
+// Get all comments
+router.get("/", async (req, res) => {
+  try {
+    const allComments = await Comment.findAll({
+      attributes: ["id", "comment_text", "post_id", "user_id", "created_at"],
+      order: [["created_at", "DESC"]],
+      include: [
+        {
+          model: User,
+          attributes: ["username"],
+        },
+        {
+          model: Post,
+          attributes: ["id", "title", "content", "created_at"],
+        },
+      ],
+    });
+    res.status(200).json(allComments);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json("Error retrieving all comments data from database.");
+  }
+});
+
+// Get one comment
+router.get("/:id", async (req, res) => {
+  try {
+    const allComments = await Comment.findByPk(req.params.id, {
+      attributes: ["id", "comment_text", "post_id", "user_id", "created_at"],
+      include: [
+        {
+          model: User,
+          attributes: ["username"],
+        },
+        {
+          model: Post,
+          attributes: ["id", "title", "content", "created_at"],
+          include: {
+            model: User,
+            attributes: ["username"],
+          },
+        },
+      ],
+    });
+    if (!allComments) {
+      res.status(404).json("No comment with this ID in the database");
+      return;
+    }
+    res.status(200).json(allComments);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json("Error retrieving comment data from database.");
+  }
+});
+
+// Create comment
+router.post("/", async (req, res) => {
+  try {
+    await Comment.create({
+      comment_text: req.body.commentText,
```

Comment:
great job assigning the vals from the body directly to the keys here! 
An alternative would've been just to pass the entire req.body but that is quite dangerous as anyone could send anything in that body and thus, potentially break your app.
Well handled!